src/test/java/com/thealgorithms/maths/FibonacciNumberTest.java

```diff
@@ -0,0 +1,17 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class FibonacciNumberTest {
+
+    private FibCalc calculator = new FibCalc();
```

Comment:
`calculateFibonacciUsingLoop` is a static method, you do not need an instance of `FibCalc` to call it.